bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-participants/list-item/component.tsx

```diff
@@ -121,7 +121,8 @@ const UserListItem: React.FC<UserListItemProps> = ({ user, lockSettings }) => {
   if (user.mobile && LABEL.mobile) {
     subs.push(intl.formatMessage(messages.mobile));
   }
-  if (user.locked && lockSettings?.hasActiveLockSetting && !user.isModerator) {
+  if ((user.locked || user.userLockSettings?.disablePublicChat)
+      && (user.userLockSettings?.disablePublicChat || lockSettings?.hasActiveLockSetting) && !user.isModerator) {
```

Comment:
It is something we need to align.
On the backend the validation is different, if the user has individual `disablePublicChat` it doesn't check if the user is moderator or not.

If moderator status should ignore this prop, we will need to tweak that validation accordingly.

Reply:
adjusting validation (https://github.com/bigbluebutton/bigbluebutton/pull/21133/commits/0a6dac83b5bb932d8f8609b830a311af9d47aab6) so disablePublicChat only affect viewers